src/components/about/About.component.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { TitleComponent } from '../title/Title.component';
+
+interface AboutProps {
+    customClasses: string;
+}
+
+export const AboutComponent: React.FC<AboutProps> = ({ customClasses }) => (
+    <article className={customClasses}>
+        <TitleComponent>About me</TitleComponent>
+        <p>
+            Hi! My name is Susanna and I&apos;m a Junior Frontend Developer. I am already familiar with main Web Technologies like React,
+            HTML, CSS, JavaScript and Git version control system.
+        </p>
+        <p>This page was developed during the course &apos;Intro to React&apos; from Masters Academy in 2024.</p>
```

Comment:
I mean, on the design this is the link to our course promo. So we should add it here with [target="_blank"](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/target) - to open in the new browser tab
```suggestion
        <p>This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it" target="_blank" >&apos;Intro to React&apos;</a> from Masters Academy in 2024.</p>
```

Reply:
done